SpacetimeWaveToyX/test/standing/standing.par

```diff
@@ -15,9 +15,9 @@ CarpetX::ncells_x = 8
 CarpetX::ncells_y = 8
 CarpetX::ncells_z = 8
 
-CarpetX::periodic_x = yes
-CarpetX::periodic_y = yes
-CarpetX::periodic_z = yes
+Driver::periodic_x = yes
```

Comment:
This doesn't enable periodic boundaries.

Reply:
no it does not, this should be Driver::periodic = yes. Hmm, I thought the tests had all passed. Will check.

Reply:
That's in the output directory. The par file should not even be there. Removed.